<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="PHPUnitTest" file="C:\Users\recca\github\tester-phpunit\tests\PHPUnitTest.php" tests="4" assertions="2" errors="0" failures="1" skipped="2" time="0.009427">
    <testcase name="testPassed" class="PHPUnitTest" classname="PHPUnitTest" file="C:\Users\recca\github\tester-phpunit\tests\PHPUnitTest.php" line="13" assertions="1" time="0.004650"/>
    <testcase name="testFailed" class="PHPUnitTest" classname="PHPUnitTest" file="C:\Users\recca\github\tester-phpunit\tests\PHPUnitTest.php" line="18" assertions="1" time="0.002471">
      <failure type="PHPUnit\Framework\ExpectationFailedException">PHPUnitTest::testFailed
Failed asserting that false is true.

C:\Users\recca\github\tester-phpunit\tests\PHPUnitTest.php:20
</failure>
    </testcase>
    <testcase name="testSkipped" class="PHPUnitTest" classname="PHPUnitTest" file="C:\Users\recca\github\tester-phpunit\tests\PHPUnitTest.php" line="23" assertions="0" time="0.001352">
      <skipped/>
    </testcase>
    <testcase name="testIncomplete" class="PHPUnitTest" classname="PHPUnitTest" file="C:\Users\recca\github\tester-phpunit\tests\PHPUnitTest.php" line="28" assertions="0" time="0.000954">
      <skipped/>
    </testcase>
  </testsuite>
</testsuites>
